src/form-data/ReviewPage.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { useContext, useEffect, useLayoutEffect } from 'react';
+import {
+  FormikBag,
+  FormikComputedProps,
+  FormikContextType,
+  FormikProps,
+  useFormikContext,
+} from 'formik';
+import { Link, useLocation } from 'react-router-dom';
+import { VaOnThisPage } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { Page, PageProps } from '../routing';
+import { FieldObject, PageObject } from './types';
+
+import { parseDate } from '../utils/helpers';
+import { PageContext } from './PageContext';
+// import { getRadioLabel } from "../utils/helpers";
+
+/**
+ * Transforms fields value into value that is more readable
+ * @param {boolean | object | string} field object value
+ * @param {string} key object key name
+ *
+ * @beta
+ */
+const transformFieldValue = (key: number, field: FieldObject) => {
+  if (typeof field?.value === 'undefined' || typeof field?.value === 'object')
+    return;
+
+  if (field?.options) {
+    const fieldIndex = field.options.findIndex(
+      (fieldOption) => fieldOption.value === field?.value
+    );
+    if (fieldIndex > 0) {
+      return field.options[fieldIndex].label;
+    }
+  }
+  if (field.value === 'true' || field.value === true) {
+    return 'Yes';
+  }
+  if (field.value === 'false' || field.value === false) {
+    return 'No';
+  }
+  if (
+    ['from', 'to', 'veteranDateOfBirth', 'deathDate', 'burialDate'].indexOf(
+      field.name
+    ) > -1
```

Comment:
Field names like these are usually specific to a form; there's no certainty that a team will have the same field names in their form as we do in ours. I wonder if there's a good way to make this a little more generic 🤔 Could we give users the option to pass in a list of their own labels to handle here? Or, could we potentially allow them to pass in their own callback that handles label customization for fields like this?